aws/resource_aws_ssm_document_test.go

```diff
@@ -108,6 +108,35 @@ func TestAccAWSSSMDocument_permission_private(t *testing.T) {
 	})
 }
 
+func TestAccAWSSSMDocument_permission_change(t *testing.T) {
+	name := acctest.RandString(10)
+	initial_ids := "123456789012,123456789013"
+	updated_ids := "123456789012"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSSSMDocumentDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSSSMDocumentPrivatePermissionConfig(name, initial_ids),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
This check should verify the value of the account IDs either via `resource.TestCheckResourceAttr("aws_ssm_document.foo", "permissions.account_ids", initialIds),` or writing a new `testAccCheck` function that verifies it from the API.